src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/ast.go

```diff
@@ -681,19 +681,100 @@ func (optList DatabaseOptionListOpt) Format(buf *TrackedBuffer) {
 	}
 }
 
-// TableOptions represents the table options.
+// TableOptionType is the type for table_options
+type TableOptionType int
+
+const (
+	TableOptionNone TableOptionType = iota
+	TableOptionComment
+	TableOptionEngine
+	TableOptionAutoInc
+	TableOptionCharset
+	TableOptionTableType
+)
+
+// TableOption represents the table options.
+// See https://dev.mysql.com/doc/refman/5.7/en/create-table.html
+type TableOption struct {
+	Type TableOptionType
+	Val  *SQLVal
+}
+
+type TableOptionList []*TableOption
+
+type TableOptionListOpt struct {
+	TblOptList TableOptionList
+}
+
+// Although each option can be appeared many times in MySQL, we make a constraint
+// that each option should only be appeared just one time in RadonDB.
+func (tblOptList *TableOptionListOpt) CheckIfTableOptDuplicate() string {
+	var optOnce = map[TableOptionType]bool{
+		TableOptionComment:   false,
+		TableOptionEngine:    false,
+		TableOptionAutoInc:   false,
+		TableOptionCharset:   false,
+		TableOptionTableType: false,
+	}
+	for _, opt := range tblOptList.TblOptList {
+		switch opt.Type {
+		case TableOptionComment:
+			if optOnce[TableOptionComment] {
+				return "Duplicate table option for keyword 'comment', the option should only be appeared just one time in RadonDB."
+			}
+			optOnce[TableOptionComment] = true
+		case TableOptionEngine:
+			if optOnce[TableOptionEngine] {
+				return "Duplicate table option for keyword 'engine', the option should only be appeared just one time in RadonDB."
+			}
+			optOnce[TableOptionEngine] = true
+		case TableOptionAutoInc:
+			if optOnce[TableOptionAutoInc] {
+				return "Duplicate table option for keyword 'auto_increment', the option should only be appeared just one time in RadonDB."
+			}
+			optOnce[TableOptionAutoInc] = true
+		case TableOptionCharset:
+			if optOnce[TableOptionCharset] {
+				return "Duplicate table option for keyword 'charset', the option should only be appeared just one time in RadonDB."
+			}
+			optOnce[TableOptionCharset] = true
+		case TableOptionTableType:
+			if optOnce[TableOptionTableType] {
+				return "Duplicate table option for keyword 'single or globle', the option should only be appeared just one time in RadonDB."
+			}
+			optOnce[TableOptionTableType] = true
+		}
+	}
+	return ""
+}
+
+func (tblOptList *TableOptionListOpt) GetTableOptValByType(optType TableOptionType) *SQLVal {
+	for _, opt := range tblOptList.TblOptList {
+		if opt.Type == optType {
+			return opt.Val
+		}
+	}
+
+	return nil
+}
+
+// TableOptions is used by TableSpec
 type TableOptions struct {
+	Comment string
 	Engine  string
+	AutoInc string // Currently not used, default "", actually it`s type is int
```

Comment:
the AutoInc has its meaning.

Reply:
> the AutoInc has its meaning.

We can remove AutoInc  option since it is useless.